src/odemis/gui/comp/legend.py

```diff
@@ -347,6 +369,13 @@ def on_paint(self, _):
 
         for i, (pos, val) in enumerate(self._tick_list):
             label = units.readable_str(val, self.unit, sig)
+
+            if i == 0 and self._lo_ellipsis:
+                label = "..." + label
```

Comment:
There is a special ellipsis character, which might display better with some fonts. So you might want to use it: `u"…"`.